### ISSUE TITLE
Jira Server support

### ISSUE THREAD

**Issue by isakar** [ncipollo/tix#47]
hey @ncipollo! Since we moved to Jira Server, tix gets an unauthorized error:
```
😱  unable to fetch jira fields
Error: 401 : 
```
It has been a nightmare creating tickets without tix 😅. Do you have any plans to support Jira Server?

**Comment by ncipollo** [comment]
Hey @isakar!

Unfortunately I don't have access to a Jira Server instance any longer so it would be kinda hard to test out functionality for that type of configuration. Also, FYI Jira Server will be end of life in 2024, so I imagine you all will probably be migrating to cloud at some point.

That being said, I did get tix to work with your Jira Server instance in the past. I just needed to use my password (maybe okta password?) as my Jira secret and it worked.

**Comment by isakar** [comment]
Ah, good to know! 
I actually tried before with my ~~git~~ jira password and didn't work, but I didn't try with the okta pass, so I will try that🤞

**Comment by ncipollo** [comment]
Yeah, it should be basically whatever you password to Jira is, if you are using a single
sign in then it will be the password for that (ex - okta).
